playwright_test/utils/Url.js

```diff
@@ -0,0 +1,3 @@
+export const URL_DETAILS = {
+    CHATWIDGET_URL: "https://test.prochnost.tk/widget-test",
```

Comment:
we can think of deploying this on one of our own domains

Reply:
Will discuss with QA team and uodate.